Dockerfile.build

```diff
@@ -0,0 +1,27 @@
+# Dockerfile.build represents the build environment of the driver, used during
+# the development phase to test and in CI to build and test.
+
+# The prefered base image is the lastest stable Alpine image, if alpine doesn't
+# meet the requirements you can switch the from to the latest stable slim
+# version of Debian (eg.: `debian:jessie-slim`).
+FROM node:7-alpine
```

Comment:
we can perfectly use the LTS, was the first that I found, tbh. Anyway, Dockerfiles are bound to change once the SDK is ready to use, so they will get changed and then we'll decide what version to use. But yeah, LTS makes sense